pyglider/utils.py

```diff
@@ -197,11 +197,13 @@ def get_profiles_new(ds, min_dp = 10.0, inversion=3., filt_time=100,
 
 
 def get_derived_eos_raw(ds):
-    # GPCTD requires a scale factor of 10 for conductivity. Legato does not
-    if 'LEGATO' in ds.conductivity.source:
+    # GPCTD and slocum ctd require a scale factor of 10 for conductivity. Legato does not
+    if 'S m' in ds.conductivity.units:
+        r = 10 * ds.conductivity / seawater.constants.c3515
+    elif 'mS cm' in ds.conductivity.units:
         r = ds.conductivity / seawater.constants.c3515
     else:
-        r = 10 * ds.conductivity / seawater.constants.c3515
+        _log.error("Could not parse conductivity units in yaml. Expected 'S m-1' or 'mS cm-1'")
```

Comment:
should this actually raise?   I don't think errors in the log raise, and if they don't then r won't be defined, and it will fail below.

Reply:
Testing it looks like this does not raise. Would it be better to raise a ValueError if it can't match either of the clauses?

Reply:
I think so with a robust error message that tells folks to edit the yml?

Reply:
Sounds good, implemented. it now raises a ValueError if it can't parse the conductivity units